### DISCUSSION TITLE
LaTex rendering in flet

### DISCUSSION THREAD

**Discussion by MohamedMoustaphaHamed** [flet-dev/flet#3955] [General]
Hello everyone. 
Is there a way to render html or IPython in flet page so that I can see latex output on the page. Please note that I try with Markdown control but it didn't work.

**Comment by Abdosql47** [comment]
I'm looking for a library I came across called flet_math   If this is helpful, we hope the developers will improve Markdown to support LaTeX, code coloring, and formatting. diagrams mermaid    
visit this : 
https://pypi.org/project/flet-math/
